requirements-dev.txt

```
@@ -2,4 +2,8 @@ sphinx-autobuild
 sphinx
 pytest
 flake8
-tox
\ No newline at end of file
+tox
+doctr; python_version >= '3.0'
```

I don't think we need developers to have doctr installed. It might be better to explicitly include these in the .travis.yml file so that they don't have to download something they don't need.
```suggestion
doctr; python_version >= '3.0'
```

---

Yes, you are right.